app/assets/stylesheets/application.css.scss.erb

```diff
@@ -27,6 +27,10 @@ body { padding-top: 60px; }
   margin-bottom: 100px;
 }
 
+table {
```

Comment:
I thought It doesn't matter even if we make the code outside of .well.description to affect the whole table, because the default table normally should style. 

But now, I think we should change the code conservatively, because maybe there are other default tables own their style, if we change the hole table, maybe it happen something that we can't expect.

So maybe we should make the code inside of .well.description this time